go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -186,6 +188,12 @@ func runOvnKube(ctx *cli.Context) error {
 		panic(err.Error())
 	}
 
+	// Set up a watch on our config file; if it changes, we exit -
+	// (we don't have the ability to dynamically reload config changes).
+	if err := watchForChanges(configFile); err != nil {
+		logrus.Fatalf("unable to setup configuration watch: %v", err)
+	}
+
```

Comment:
Shouldn't we move Lines 191-196 .. within the `if-arm` at L218? We need to monitor changes to the file only when we are running in `master` or `worker` mode and not when we running transient commands like `cleanup-node`?

Reply:
Move code as suggested.